operator/operator.go

```diff
@@ -41,16 +41,6 @@ func ProcessCreate(obj interface{}, operator Operator) error {
 		return microerror.MaskAny(err)
 	}
 
-	deleteState, err := operator.GetDeleteState(obj, currentState, desiredState)
-	if err != nil {
-		return microerror.MaskAny(err)
-	}
-
-	err = operator.ProcessDeleteState(obj, deleteState)
-	if err != nil {
-		return microerror.MaskAny(err)
-	}
-
```

Comment:
This was fun. I was super convinced by that but reality has proven my wrong. It does not work as I suggested in the first place. The deletion logic would have to be different based on the the event type (create vs. delete). So any eventual deletion cannot just simply be done during the create event. We do not have a real need for this at the moment. So we can still think about this over time.